build.gradle

```diff
@@ -22,7 +22,7 @@ group = 'com.aoe.jenkins'
 def generatedSourcesDir = 'generated'
 
 jenkinsPlugin {
-    coreVersion = '1.600'
+    coreVersion = '2.46.1'
```

Comment:
Is this necessary? It makes the plugin unusable for all Jenkins instances with version < '2.46.1'. At least it should be the minimal version needed.